GageStatsDB/Resources/Agency.cs

```diff
@@ -30,10 +30,11 @@ public partial class Agency
         public int ID { get; set; }
         [Required]
         public string Name { get; set; }
+        [Required]
```

Comment:
I think this is fine, however this will only enforce the field to be required at the services level not the database, unless you push the changes to the database using the ef dotnetcore cli migrations. Like I mentioned, its not a big deal, this comment is mostly just for informational purposes